core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -140,6 +140,10 @@ private TableProperties() {
   public static final String WRITE_DISTRIBUTION_HASH_AUTO = "auto";
   public static final String WRITE_DISTRIBUTION_HASH_DEFAULT = WRITE_DISTRIBUTION_HASH_AUTO;
 
+  public static final String WRITE_OUTPUT_FILE_FACTORY = "write.output.file.factory.impl";
```

Comment:
`write.out-put-file-factory-impl` is better.

Reply:
ok, but don't need this property now, I will remove this property